.circleci/config.yml

```diff
@@ -159,6 +159,21 @@ jobs:
             pegasus_port: 3000
             " >> locals.yml
 
+      - restore_cache:
+          keys:
+            - code-dot-org-ui-test-data-v3-staging-
```

Comment:
why is there a trailing - in this key?

Reply:
Circle does a prefix match if there's no exact match, and in this case there will never be an exact match since I'm appending the commit hash to the key. Leaving the '-' on there just reminds the reader that it's a prefix, not a complete key.